src/data-tutorials/users.tsx

```diff
@@ -853,18 +864,21 @@ const Users: User[] = [
     tags: ['advanced', 'foundation', 'crowdfunding', 'composer'],
   },
   {
-    title: 'React Native & Celo - Easily build React Native dApps on Celo',
-    description: 'Quickly develop Android and iOS apps on Celo using the Celo Composer for React Native.',
-    preview: require('./showcase/intermediate/easily-build-react-native-dapps-on-celo.png'),
-    website: 'blog/tutorials/React-Native-&-Celo-Easily-build-React-Native-dApps-on-Celo',
-    tags: ['intermediate','foundation', 'reactnative', 'composer'],
```

Comment:
Don't change the single quotes to double quotes

Reply:
Hello @developerkunal, I don't seem to get the problem. Can you make the edit on my behalf?